coap-gateway/service/client.go

```diff
@@ -834,3 +834,10 @@ func (client *Client) CreateResource(ctx context.Context, event *events.Resource
 
 	return nil
 }
+
+func (client *Client) OnDeviceSubscriberReconnectError(err error) {
+	auth, _ := client.GetAuthorizationContext()
+	deviceID := auth.GetDeviceID()
+	log.Errorf("error occurs during reconnect device(%v) subscriber: %v", deviceID, err)
```

Comment:
Could you please explain when can the error occur? This message is not clear enough.
during reconnect device xyz subscriber abc.

Reply:
fixed